File: FirstBit Workspace/C Programming/Assignments/Assignment 8/Q7.c
/*Take two array and add sum in third array

Example-
arr[5]= {1,2, 3, 4,5}
brr[5]={10,20,30, 40, 50}
crr[5]={11,22,33,44,55}*/

#include<stdio.h>
void main(){
	
	int arr[5], brr[5], crr[5];
	
	//storing numbers in 1st array
	printf("Storing elements for 1st array");
	printf("\nEnter 5 numbers to store in a array: ");
	for(int i=0;i<5;i++)
		scanf("%d",&arr[i]);
		
	//storing numbers in 2nd array
	printf("\n\nStoring elements for 2nd array");
	printf("\nEnter 5 numbers to store in a array: ");
	for(int i=0;i<5;i++)
		scanf("%d",&brr[i]);
		
	//printing 2 arrays
	printf("\nFirst array: ");
	for(int i=0;i<5;i++)
		printf("%d ",arr[i]);
		
	printf("\nSecond array: ");
	for(int i=0;i<5;i++)
		printf("%d ",brr[i]);
		
	//sum in third array
	printf("\n\nThird array: ");
	for(int i=0;i<5;i++){
		crr[i]=arr[i]+brr[i];
		printf("%d ",crr[i]);
	}
		
}

File: FirstBit Workspace/Core Java/In Class Work/ArrayCProgramsInJava/src/Q7.java
import java.util.Scanner;

public class Q7 {

	public static void main(String[] args) {
		
		int[] arr = new int[5];
		int[] brr = new int[5];
		int[] crr = new int[5];
				
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Enter element to store in array1: ");
		for(int i=0; i<arr.length; i++) {
			arr[i] = sc.nextInt();
		}
		
		System.out.println("\nEnter element to store in array2: ");
		for(int i=0; i<brr.length; i++) {
			brr[i] = sc.nextInt();
		}
		
		//display elements from array
		System.out.print("\narr[5] = {");
		for(int i=0; i<arr.length; i++) {
			
			if(i == arr.length - 1)
				System.out.print(arr[i]);
			else 
				System.out.print(arr[i] + ", ");
			
		}
		System.out.println("}");
		
		System.out.print("brr[5] = {");
		for(int i=0; i<brr.length; i++) {
			
			if(i == brr.length - 1)
				System.out.print(brr[i]);
			else 
				System.out.print(brr[i] + ", ");
			
		}
		System.out.println("}");
		
		//Add Two Arrays and Store Result in Third Array
		System.out.print("crr[5] = {");
		for(int i=0; i<crr.length; i++) {
			crr[i] = arr[i] + brr[i];
			if(i == crr.length - 1)
				System.out.print(crr[i]);
			else 
				System.out.print(crr[i] + ", ");
		}
		System.out.println("}");
		
		System.out.println("\nResult array: ");
		for(int ele:crr) {
			System.out.println(ele);
		}
		
		sc.close();

	}

}
